src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -477,11 +476,13 @@ public double getAngleTowardsCenterOfMass() {
 						+ INNER_ARM_LENGTH * Math.sin(innerArmTheta);
 
 		double wristCenterOfMassX =
-				WRIST_CENTER_OF_MASS_DISTANCE_FROM_JOINT * Math.cos(wristTheta)
+				WRIST_CENTER_OF_MASS_DISTANCE_FROM_JOINT
+								* Math.cos(wristTheta + WRIST_CENTER_OF_MASS_POSITION_OFFSET)
```

Comment:
Not sure if the units are right here, `wristTheta` should be in radians whereas `WRIST_CENTER_OF_MASS_POSITION_OFFSET` seems to be in rotations.